src/test/config/bitbucket/values-EKS.yaml

```diff
@@ -5,7 +5,15 @@ serviceAccount:
 
 bitbucket:
   ingress:
-    fqdn: ${helm.release.prefix}-bitbucket.${eks.ingress.domain}
+    create: true
+    host: ${helm.release.prefix}-bitbucket.${eks.ingress.domain}
+    annotations:
+      "kubernetes.io/ingress.class": "nginx"
+      "nginx.ingress.kubernetes.io/affinity": "cookie"
+      "nginx.ingress.kubernetes.io/affinity-mode": "persistent"
+      "nginx.ingress.kubernetes.io/proxy-connect-timeout": "60"
+      "nginx.ingress.kubernetes.io/proxy-read-timeout": "60"
+      "nginx.ingress.kubernetes.io/proxy-send-timeout": "60"
```

Comment:
These annotations are going to be common if the customer uses the de-facto standard kube-nginx ingress controller, so in a later PR I'll move these into the main charts.